task5/src/check.java

```diff
@@ -0,0 +1,19 @@
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class Check extends HttpServlet {
```

Comment:
Incorrect lab is not working :)
Name of java file should be the same as main class name (+case)